jetstream/cli.py

```diff
@@ -94,8 +108,15 @@ def execute(
         if configuration_map is not None:
             raise Exception("Custom configurations are not supported when running with Argo")
 
-        experiments_config = [
-            {"date": date.strftime("%Y-%m-%d"), "slug": slug} for (slug, date) in worklist
+        experiments_config: Dict[str, List[str]] = {}
+        for (slug, date) in worklist:
+            if slug not in experiments_config:
+                experiments_config[slug] = []
+
+            experiments_config[slug].append(date.strftime("%Y-%m-%d"))
+
+        experiments_config_list = [
+            {"slug": slug, "dates": dates} for slug, dates in experiments_config.items()
```

Comment:
I needed to change this data structure that is passed into the `run.yaml` Argo workflow definition. This will allow Argo to run the ensure-enrollments task followed by analysis tasks for different experiments in parallel. (See workflow file changes below)